export/exporter.go

```diff
@@ -160,8 +160,14 @@ func (e *exporter) exportPkg(p *packages.Package, proID string) (err error) {
 		if err = e.exportDefs(p, f, fi, proID, fpos.Filename); err != nil {
 			return fmt.Errorf("error exporting definitions of %q: %v", p.PkgPath, err)
 		}
+	}
 
-		if err := e.exportUses(p, fi, fpos.Filename); err != nil {
+	// NOTE: since we currently only support package-level references, it is OK to run the loop
+	//       here. Once we want repository (or Go module)-level references, they can only be
+	//       handled after all files are processed for definitions.
```

Comment:
How about s/run the loop here/export usages at the end of each package/ and "When repository-level references are implemented, usages must be exported after all files are processed"? I'm not 100% sure if that's correct.

Reply:
I think your description is much better!